src/components/checkbox.js

```diff
@@ -27,8 +27,13 @@ export default class Checkbox extends EventEmitter {
             this.emit('initialized', this.input.checked)
         })
 
-        this.input.onchange = (data) => {
```

Comment:
i did not want to make all checkbox controls to listen on space because in my project i have a global eventlistener to listen for space and start/stop animation. so this is going to be reverted. maybe we can have a setting for that later.